### PR TITLE
Refactor the ContainerQueryRunner to utilize the JDBC driver for executing queries

### PR DESCRIPTION
**Pull request by Joe-Abraham**
## Description
Refactor the ContainerQueryRunner to utilize the JDBC driver for executing queries.

### CODE REVIEW

**Review by tdcmeehan — COMMENTED** [review]
Directionally this looks fine.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunner.java [hunk 1]
```diff
@@ -276,34 +296,14 @@ public Session getDefaultSession()
...
+                    .toMaterializedResult(
+                            statement.executeQuery(sql));
         }
-        catch (IOException e) {
-            throw new UncheckedIOException(e);
-        }
-        catch (InterruptedException e) {
-            throw new RuntimeException(e);
+        catch (SQLException e) {
+            e.printStackTrace();
```
Comment (tdcmeehan) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunner.java:305]: Please throw

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java [hunk 2]
```diff
@@ -224,92 +236,169 @@ public static void createScriptFile(String filePath, String scriptContent)
...
+                return TimestampType.TIMESTAMP;
+            case "TIMESTAMP WITH TIME ZONE":
+                return TimestampWithTimeZoneType.TIMESTAMP_WITH_TIME_ZONE;
+            case "VARBINARY":
+            case "BINARY":
+                return VarbinaryType.VARBINARY;
+            default:
+                // Log or handle unknown type names
+                System.out.println("Unknown SQL type name: " + typeName);
+                return UnknownType.UNKNOWN;
```
Comment (tdcmeehan) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java:401]: Throw an error here

**Review by tdcmeehan — CHANGES_REQUESTED** [review]

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java (hunk 2 above)
Comment (Joe-Abraham) [reply presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java]: Updated the code.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunner.java [hunk 3]
```diff
@@ -95,7 +98,28 @@ public ContainerQueryRunner(int coordinatorPort, String catalog, String schema,
...
+                coordinator.getMappedPort(coordinatorPort),
+                catalog,
+                schema,
+                "timeZoneId=UTC");
+
+        try {
+            Connection connection = DriverManager.getConnection(url, "test", null);
+            statement = connection.createStatement();
+        }
+        catch (Exception e) {
```
Comment (tdcmeehan) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunner.java:114]: What is thrown here?  If it's an IOException, can we let it simply throw?

Comment (Joe-Abraham) [reply presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunner.java:114]: It is an SQLException.

**Comment by tdcmeehan** [comment]
Commit message is too long.  Perhaps `Use JDBC for ContainerQueryRunner`.

**Comment by Joe-Abraham** [comment]
Updated the commit message

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**